ttkd-admin_panel/src/App.css

```diff
@@ -2,7 +2,7 @@
   text-align: center;
 }
 
-.App-logo {
+/*.App-logo {
```

Comment:
Is this commented out with an intent to repurpose it later? If not, we could probably just delete it